src/config/actionTriggers.ts

```diff
@@ -0,0 +1,21 @@
+import { ActionTriggers } from '@graasp/sdk';
+
+export const actionsDescriptionTransKeys: { [key: string]: string } = {
```

Comment:
If you want to be a bit tighter:
```suggestion
export const actionsDescriptionTransKeys: { [key: `${ActionTriggers}`]: string } = {
```